src/main/java/no/digipost/signature/client/domain/SignatureJobReference.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) Posten Norge AS
```

Comment:
Adding missing headers is a matter of running `mvn license:format`. I don't know about legal stuff, but maybe @landro has any insight?